modules/channel-web/src/views/lite/messages/index.jsx

```diff
@@ -339,18 +339,11 @@ class Message extends Component {
     delete messageDataProps.component
 
     const props = {
-      onSendData: this.props.onSendData,
+      ..._.pick(this.props, ['isLastGroup', 'isLastOfGroup', 'onSendData']),
```

Comment:
sorry my misunderstanding, what's that ?

Reply:
those are used to know if it's the last message in the conversation (and the last one in a group, let's say you send multiple messages).

When you speak with the bot, it's no longer the last, then you can hide the component